gui/serial_unix.cc

```diff
@@ -90,15 +90,15 @@ static const char* deviceNames[] = {
   "/dev/ttyS3",
   "/dev/ttyUSB0",
   "/dev/rfcomm0",
-  0
+  nullptr
```

Comment:
How is this code still here? Didn't we move to QSerialPortInfo::availablePort years ago? (I know this CL was cargo-culted and not introspecting the origins, so not an objection)